base/shell/explorer/traywnd.cpp

```diff
@@ -200,8 +200,10 @@ class CStartButton
 
     VOID Initialize()
     {
-        // HACK & FIXME: CORE-17505
-        SubclassWindow(m_hWnd);
+        // HACK & FIXME
+        HWND hWnd = m_hWnd;
+        m_hWnd = NULL;
+        SubclassWindow(hWnd);
```

Comment:
Kind of unnatural (here and in the other places). What does the SubclassWindow function expect from `m_hWnd` ?

Reply:
`SubclassWindow` does assert `m_hWnd == NULL`.